code/datums/components/happiness.dm

```diff
@@ -56,7 +56,7 @@
 	ADD_TRAIT(parent, TRAIT_MOB_RELAY_HAPPINESS, REF(src))
 
 /datum/component/happiness/UnregisterFromParent()
-	UnregisterSignal(parent, list(COMSIG_HOSTILE_PRE_ATTACKINGTARGET, COMSIG_COMPONENT_CLEAN_ACT, COMSIG_MOB_ATE))
+	UnregisterSignal(parent, list(COMSIG_HOSTILE_PRE_ATTACKINGTARGET, COMSIG_COMPONENT_CLEAN_ACT, COMSIG_MOB_ATE, COMSIG_LIVING_DEATH, COMSIG_LIVING_REVIVE))
```

Comment:
Is this an artifact of an earlier version of your change? It doesn't look like those signals are registered.

Reply:
You are right, I tried to make their happiness pause on death and resume on life, but I decided against it, and did not fully clean it up. I will fix this today.